CHANGELOG.md

```diff
@@ -1,3 +1,60 @@
+# [4.0.0-beta.3](https://github.com/algolia/vue-instantsearch/compare/v4.0.0-beta.2...v4.0.0-beta.3) (2021-08-20)
```

Comment:
should we change this to just say v4 and have all changes to v3?

Reply:
Yes, I'm planning to update it in a release PR for 4.0.0.